debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -410,6 +410,14 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                            .withDependents(DATABASE_WHITELIST_NAME)
                                                            .withDescription("Flag specifying whether built-in tables should be ignored.");
 
+    public static final Field JDBC_DRIVER = Field.create("database.jdbc.driver")
+            .withDisplayName("Jdbc Driver Class Name")
+            .withType(Type.STRING)
```

Comment:
That's what `Type.CLASS` does ... it simply means `Type.STRING` except the Confluent UI knows/treats it as a Java class name. It's a minimal validation when the UI user is entering values, and we still do proper validation via the `.withValidation(Field::isClassName)` call a few lines below this one.